.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
```

Comment:
Not sure for this. We have not defined the vscode rules for our projects? Why this one?

Reply:
This should be in `.gitignore`.